sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/mathExpressions.scala

```diff
@@ -170,29 +193,29 @@ case class Pi() extends LeafMathExpression(math.Pi, "PI")
 
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(expr) - Returns the inverse cosine (a.k.a. arccosine) of `expr` if -1<=`expr`<=1 or NaN otherwise.",
+  usage = "_FUNC_(expr) - Returns the inverse cosine (a.k.a. arccosine) of `expr` if -1<=`expr`<=1 or NULL otherwise.",
   extended = """
     Examples:
       > SELECT _FUNC_(1);
        0.0
       > SELECT _FUNC_(2);
-       NaN
+       NULL
```

Comment:
NaN represents a variety of meanings, and Null is a special case, then we need to modify?
Personally think that may not be necessary. @viirya do you think?